deploying.md

```diff
@@ -0,0 +1,75 @@
+## Build for Production
+
+To run a production-ready instance you check out the code as above and have the prerequisite tools installed.
+You have run the 'npm run install' set up script.
+
+Step 0. Set up the site specific environment
+```
+cd /your/development/area
+cd deployment
+cp sample.env.site .env.site
+vi .env.site
+```
+
+Make sure to modify the ``.env`` file so that it contains the correct values for your site.
+
+
+Step 1. Build and publish the client vue project.
+```
+cd /your/development/area
+cd client
+npm run both
+```
+
+Step 2. Start the docker system
+```
+cd /your/development/area
+cd deployment
+# to build the containers and start attached to see debug logs
+npm run prod:build
+# to run containers that were built in detached mode
+npm run prod:run
+# to stop detached running containers
+npm run prod:stop
+```
+
+
+## EHR screen generation
```

Comment:
We might create another .md file to cover the topic of code generation.  See the ehr-definitions directory and how the /makeEhrV2/deploy.sh script copies the ehr-definition content to both client and server.

Yet, in deployment we should mention that consistently running makeEhrV2/copy-common-src.sh and running tests in /ehr-definitions is necessary. 

ehr-definitions was created to hold code common to both client and server.  There remains some code in the client with similar functionality that needs to be replaced by the code nin ehr-definitions.

Reply:
As well, currently there is no active unit testing of the client side yet we need extensive testing of EHR content generation. Hence the separate directory with its own unit tests.
This is a good area for new contributors to review and write more tests for.